src/lib.rs

```diff
@@ -204,6 +204,15 @@ unsafe impl raw_window_handle::HasRawWindowHandle for PseudoRwh {
     }
 }
 
+#[no_mangle]
+pub extern "C" fn wgpuCreateInstance(
+    _: *const native::WGPUInstanceDescriptor
+) -> native::WGPUInstance {
+    // Rationale: See https://github.com/gfx-rs/wgpu-native/issues/116
+    // Because WGPUInstance is an opaque type this library controls and does not define the contents of, this is safe.
+    8 as native::WGPUInstance
```

Comment:
but.. why 8? :)

Reply:
I don't know if Rust checks pointer alignments... and I don't want to find out.
So I picked 8. Otherwise, 1 would be the most appropriate choice.
It doesn't really matter in any case - it must be non-zero in case an application tests the return value, that is all.